ubuntu_bridge_network_configuration.sh

```diff
@@ -6,75 +6,26 @@ if [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
      # Adding an IP address in the libvirt definition for this network results in
      # dnsmasq being run, we don't want that as we have our own dnsmasq, so set
      # the IP address here
-     IFCFG_PROVISIONING_FILE=/etc/netplan/provision-bridge.yaml
-     IFCFG_PROVISIONING=$(cat << EOF
-network:
-  version: 2
-  renderer: networkd
-  bridges:
-    provisioning:
-      dhcp4: no
-      addresses: [172.22.0.1/24] 
-      interfaces: []
-EOF
-     )
+     sudo brctl addbr provisioning
+     sudo ifconfig provisioning 172.22.0.1 netmask 255.255.255.0 up 
```

Comment:
use ip command. ifconfig commands are deprecated now.

Reply:
Yes, ifconfig commands are deprecated on Ubuntu now, so I added `net-tools` package.

Reply:
I have worked on this one! Got it working, but I think I do another pull request when this on is pulled to upstream?